### PR TITLE
Implement centre bar lights and HID support

### LINKED ISSUE #8: Add PWAN center bar lighting

### PR DESCRIPTION
**Pull request by ASleepyCat**
Closes #8.

* Only supports HID lighting
* The light bar mode switch uses B10 since it's only available on PWANs

### CODE REVIEW

**Comment by ASleepyCat** [comment]
Playtested and no problems. It would be nice to have a reference to the LM cab since I'm not entirely sure that the colours are right.

File: fw/beef.cpp [hunk 1]
```diff
@@ -322,7 +317,8 @@ void process_combos(config* current_config,
     ignore_combo = false;
     timer_init(combo_timer);
     timer_init(combo_lights_timer);
-    timer_init(combo_tt_led_timer);
+    timer_init(&RgbManager::Turntable::combo_timer);
+    timer_init(&RgbManager::Bar::combo_timer);
```
Comment (HWXLR8) [inline fw/beef.cpp:321]: Why do we need separate timers for each combo? Is there ever a case when more than one timer will be running at once?

**Review by HWXLR8 — APPROVED** [review]
Looks good, will test on my PWAN shortly

File: fw/beef.cpp (hunk 1 above)
Comment (ASleepyCat) [reply fw/beef.cpp]: The rationale was that there might be a combo that would apply an effect to the bar, but I guess we can remove it and just re-add it when it's needed.

**Pull request merged**